src/search_unused.rs

```diff
@@ -146,10 +145,23 @@ fn collect_paths(dir_path: &Path, analysis: &PackageAnalysis) -> Vec<PathBuf> {
         trace!("adding src/ since paths was empty");
     }
 
+    // Unfortunately WalkBuilder does not implement `std::iter::FromIterator`.
+    let walk_builder = {
+        let mut roots = root_paths.into_iter().map(|p| dir_path.join(p));
+        let builder = ignore::WalkBuilder::new(
```

Comment:
For what it's worth: I think it's fine to keep `walkdir` among the dependencies, since it seems to be used by another transitive dependency (as it has not been removed from the `Cargo.lock` file), so we could spare the change in this function (and file).

Reply:
That's right, `walkdir` is used by `ignore`.